mongo/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - mongo
 
+## 1.6.0 / 2018-06-13
+
+* [Added] Allow disabling of replica access. See [#1516](https://github.com/DataDog/integrations-core/pull/1516).
+* [Changed] Properly parse metric. See [#1498](https://github.com/DataDog/integrations-core/pull/1498).
```

Comment:
Hey @ofek, do you mind making this PR title a little bit more customer friendly?